<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewFeedbackReportDialog</class>
 <widget class="QDialog" name="ViewFeedbackReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 192, 196);</string>
  </property>
  <widget class="QPushButton" name="reviewed_pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 249, 249);</string>
   </property>
   <property name="text">
    <string>FEEDBACK REPORT REVIEWED</string>
   </property>
  </widget>
  <widget class="QLabel" name="taskCreated_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Feedback Report Submitted:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="reportFeedback_plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro Light</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="plainText">
    <string>REPORT FEEDBACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="reportSubmittedDateTime_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>datetime</string>
   </property>
  </widget>
  <widget class="QLabel" name="reportSubject_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>REPORT SUBJECT</string>
   </property>
  </widget>
  <widget class="QLabel" name="reachableEmailAddress_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro Light</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Author Reachable Email Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="reachableEmailAddressText_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI Semilight</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Email Address Text</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
